newdle/models.py

```diff
@@ -0,0 +1,79 @@
+from pytz import timezone, utc
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.schema import CheckConstraint
+
+from newdle.core.db import db
+from newdle.core.util import UTCDateTime, format_dt, parse_dt
+
+
+class Newdle(db.Model):
+    __tablename__ = 'newdles'
+
+    id = db.Column(db.Integer, primary_key=True)
+    creator_uid = db.Column(db.String, nullable=False, index=True)
+    title = db.Column(db.String, nullable=False)
+    duration = db.Column(db.Integer, nullable=False)
+    _timezone = db.Column('timezone', db.String, nullable=False)
+    _time_slots = db.Column('time_slots', JSONB, nullable=False)
+    final_dt = db.Column(UTCDateTime, nullable=True)
+
+    # relationship backrefs:
```

Comment:
what about using explicit relationships on both sites (using `back_populates`) instead?

Reply:
Had forgotten that exists, thanks!